include-bird

```diff
@@ -0,0 +1 @@
+path-include=/usr/share/doc/bird2/*
```

Comment:
Where is this file referred from...?

Reply:
`/etc/dpkg/dpkg.cfg.d/excludes` filters out `/usr/share/doc`. 
Here including ` /usr/share/doc/bird2/*` enable to create `/usr/share/doc/bird2/examples/bird.conf` during `apt install bird2`.
ref: https://bird.network.cz/pipermail/bird-users/2019-December/014075.html